src/widgets/block.rs

```diff
@@ -38,15 +38,15 @@ pub use title::{Position, Title};
 /// # Constructor methods
 ///
 /// - [`Block::new`] creates a new [`Block`] with no border or paddings.
+/// - [`Block::border_style`] Defines the style of the borders.
+/// - [`Block::border_type`] Sets the symbols used to display the border (e.g. single line, double
+///   line, thick or rounded borders).
```

Comment:
Can you please move these lines back down - my comment was made in error. I meant only the `bordered` function to be move - apologies.

Reply:
Yes of course !